examples/platform/silabs/SiWx917/FreeRTOSConfig.h

```diff
@@ -0,0 +1,261 @@
+/*
```

Comment:
I compared 

examples/platform/silabs/SiWx917/FreeRTOSConfig.h and examples/platform/silabs/FreeRTOSConfig.h

We still have a lot of differences .

If you want to have FreeRTOSConfig.h then we have to use lot of #ifdef

Reply:
I just compared. There is really not that many difference. Most of them already have #if around them. and the few left you should apply what we already use.

`#define configUSE_IDLE_HOOK (0) ->(1)`. We already have the handler for this, why not use it and set it at 1 like we previously did

`#define configTICK_RATE_HZ ` we can validate but we had confirmation from the platform team that we should use 1024.

last this would be `#define configPRIO_BITS 6 /* 6 priority levels. */` which I am not sure why you would need to set priority level to 6. It should change between our 2 platforms. But if there is a good reason it can be in a ifdef.. 

The rest is 100 duplication. This needs to be addressed or it is a move backwards in our cleanup work

Reply:
done

Reply:
ok removed examples/platform/silabs/SiWx917/FreeRTOSConfig.h 

And added the necessary changes inside #ifdef .